### DISCUSSION TITLE
uupsUpgradeable.sol doesn't have the upgradeTo function

### DISCUSSION THREAD

**Discussion by ighodaro-daniel** [Cyfrin/foundry-full-course-cu#2399] [Q&A]
in the lesson upgradeable smart contract the uupsUpgradeable abstract contract does not have the upgradeTo function , what function should we use then in upgrading the boxv1 to boxv2 in the upgradeBox.s.sol contract

**Comment by EngrPips** [comment]
Here sir https://github.com/Cyfrin/foundry-full-course-cu/blob/main/chronological-updates.md

<img width="1440" alt="Screenshot 2024-09-05 at 11 16 08" src="https://github.com/user-attachments/assets/6277b91d-2d1f-4c99-8054-732989b77027">
